gradle-diff-task-dependencies/plugin/src/main/groovy/com/novoda/gradle/diffdependencytask/ConditionalDependencyEvaluator.groovy

```diff
@@ -0,0 +1,39 @@
+package com.novoda.gradle.diffdependencytask
+
+import java.util.regex.Pattern
+
+class ConditionalDependencyEvaluator implements GroovyCallable<Void> {
```

Comment:
It's simply my noobiness at Groovy 😆. Is there an `Action` equivalent in Groovy? The interface is there so that it can be tested with any implementation.